src/aes.rs

```diff
@@ -69,6 +71,32 @@ pub fn decrypt(encrypted_data: &[u8], key: &H256, iv: &u128) -> Result<Vec<u8>,
     Ok(final_result)
 }
 
+// AES-256/GCM/Pkcs encryption.
+pub fn encrypt_gcm_siv(data: &[u8], key: &H256, nonce: &u128) -> Result<Vec<u8>, Error> {
+    let generic_key = GenericArray::clone_from_slice(key);
+    let aead = Aes256GcmSiv::new(generic_key);
+
+    let temp_nonce = &nonce.to_be_bytes();
+    let nonce = GenericArray::from_slice(&temp_nonce[0..12]);
```

Comment:
Would you explain why do you take only 12 bytes from the nonce?

Reply:
@sgkim126 When using AES-GCM, a 96-bit IV is generally recommended. I refer to the following [a answer](https://crypto.stackexchange.com/questions/41601/aes-gcm-recommended-iv-size-why-12-bytes). I think there is no problem with using only 12 bytes out of 16 bytes because we are using a fixed nonce.

Reply:
If you will use only 12 bytes, IMHO, you should change the type of a nonce parameter as `[u8: 12]`. Discarding information inside a function may make the user misuse.

Reply:
Also, nonce reuse is not recommended in `AES-GCM-SIV` so we should not use a fixed nonce anymore.